TarSCM/changes.py

```diff
@@ -202,6 +202,7 @@ def write_changes(self, changes_filename, changes, version, author):
         shutil.move(tmp_fp.name, changes_filename)
 
     def get_changesauthor(self, args):
+        changesauthor = None
```

Comment:
This looks like a valid bugfix for me, but a bit out of scope of the rest of the commit, correct?

Reply:
Exactly. I did have to fix this in order to successfully test my changes. Perhaps I should have made this a separate fix?

Reply:
Not for me